### DISCUSSION TITLE
For those without Photoshop.

### DISCUSSION THREAD

**Discussion by geekm0nkey** [mathoudebine/turing-smart-screen-python#387] [Themes]
Came across this the other day and was very impressed with it. So I figured I would share it here for all to use.  This is for those who do not have Photoshop nor want to sail the 7 Seas if you get my meaning. You can now use "photoshop" by going to the website http://photopea.com It is nearly a 1:1 clone of Photoshop on a website. Even has content-aware fill. I would use a good ad blocker as the site does have a banner section on the right that is annoying unless you block them all. The price you pay for free I guess.

**Comment by DarkPhoinix** [comment]
https://www.gimp.org/

**Reply by geekm0nkey** [reply]
Gimp's been around quite a while. But requires an install. 
For those who want to install something.

**Comment by amiramir2124** [comment]
https://khanephotoshop.com/

photoshop tutorials
and
Photoshop products
